velox/docs/functions/presto/window.rst

```diff
@@ -153,6 +154,7 @@ is returned.
 
 An optional ``IGNORE NULLS`` suffix can be specified after all the params, if
 ``IGNORE NULLS`` is specified, ``null`` values are ignored during offset counting.
+If all values are ``null`` during offset counting, ``default_value`` is returned.
```

Comment:
Nit : The scenario is more of non-null rows were not found vs all values are null